.eslintrc

```diff
@@ -1,12 +1,21 @@
 {
-  extends: [
-    "prettier",
-    "prettier/react",
-    "prettier/standard"
-  ],
-  parser: "babel-eslint",
-  plugins: ["react"],
-  settings: {
-    "import/resolver": "webpack",
+  "parserOptions": {
+    "ecmaVersion": 2017
+  },
+  "env": {
+    "node": true
   },
+  "extends": [
+    "eslint:recommended",
+    "plugin:jest/recommended",
+    "prettier"
+  ],
+  "plugins": [
+    "prettier"
+  ],
+  "rules": {
+    "prettier/prettier": [
```

Comment:
Since you use this plugin, you might not need running `prettier --write` in lint-staged at all. just `eslint --fix`

Reply:
good catch. thx.

Reply:
I'd advice against that plugin: it's very annoying when you develop, because half of your code is marked as error, but it's just a formatting "issue" that can be autofixed. But you don't want to be distracted when you write code.